### PR TITLE
Add post announcing project director nominees

### PR DESCRIPTION
**Pull request by eholk**
Getting this ready now. The plan is to go live on Friday, September 22.

### CODE REVIEW

**Comment by eholk** [comment]
@rust-lang/leadership-council - I've put up a draft for a post that I plan to publish Friday that announces the Rust Project Directors nominees. Please let me know if you have any suggestions or concerns with the announcement. Some of the wording could probably be better, so I'm happy to take suggestions!

File: posts/inside-rust/2023-09-22-project-director-nominees.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- Michael Goulet
+- Santiago Pastorino
+- Jubilee
+- Mara Bos
+
+These were all spoken of highly by their nominating teams.
+While several other candidates were suggested, not everyone who was nominated felt like they were in a place where they were able or wanted to accept the role if they were elected.
+
+Now we'd again like to ask the members of the Rust Project for their feedback.
+Please let us know if there's anything we should consider going into the election process.
```
Comment (Mark-Simulacrum) [inline posts/inside-rust/2023-09-22-project-director-nominees.md:30]: Let us know - can we be more specific? For example, email / Zulip / specific people contacted... I think in particular this may be relevant for Mara as representative and nominee, which could be a conflict of interest for that set of teams (at least potentially).

**Comment by thejpster** [comment]
I don't know how people would find a list of Zulip handles or email addresses for the council members.

We should address the conflict with Mara up front and explain how we deal with that in a fair fashion.

File: posts/inside-rust/2023-09-22-project-director-nominees.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+- Jakob Degen
+- Amanieu
+- Felix Klock
+- lcnr
+- scottmcm
+- Michael Goulet
+- Santiago Pastorino
+- Jubilee
+- Mara Bos
```
Comment (Turbo87) [inline posts/inside-rust/2023-09-22-project-director-nominees.md:24]: it might make sense to make these links to e.g. GitHub or other notable profiles :)

Comment (eholk) [reply posts/inside-rust/2023-09-22-project-director-nominees.md:24]: Agreed! Github is probably a good common denominator.

Going forward, I wonder if we should have some kind of project directory that can serve as a canonical place to link when we need to reference people?

Comment (eholk) [reply posts/inside-rust/2023-09-22-project-director-nominees.md]: I just pushed an update that includes GH links for everyone.

File: posts/inside-rust/2023-09-22-project-director-nominees.md (hunk 1 above)
Comment (eholk) [reply posts/inside-rust/2023-09-22-project-director-nominees.md]: I tried to rephrase this to be a little more specific. We did have a link to the council Zulip and email address down below, but I can move it up higher if that would make it more obvious.

I also added a paragraph about Mara's conflict of interest and how we will address that.

**Comment by eholk** [comment]
@thejpster 
> I don't know how people would find a list of Zulip handles or email addresses for the council members.

Good point. I took out the instructions to contact us individual by Zulip or email and instead replaced it with a generic "reach out to your team's representative." Hopefully we know how to find and contact our team's representative :)

I thought about listing everyone's Zulip handles and emails here, but I would rather not have two lists of 9-10 people in the same post. One possibility is to link to the [Leadership Council page](https://www.rust-lang.org/governance/teams/leadership-council) on rust-lang.org with a note that you can find contact information for individual council members there. The problem is, that only links to people's GitHub profiles, and not all of us have contact information on the profiles. Do you think linking to the Council page would still be valuable?

Going forward, we probably should look for ways to have clearer contact information for members of the Rust project...

> We should address the conflict with Mara up front and explain how we deal with that in a fair fashion.

I added a note about that in https://github.com/rust-lang/blog.rust-lang.org/pull/1145/commits/a8d6738ade8289b0543e07cbce4e0ae9afa73338. Does that seem sufficient?

**Comment by eholk** [comment]
@JakobDegen @Amanieu @pnkfelix @lcnr @scottmcm @compiler-errors @spastorino @workingjubilee @m-ou-se 

I'm getting a blog post ready to announce the Project Director nominees. Since you've all been nominated, I want to make sure I've listed your name and contact link in a way you're happy with. Please take a look and comment if it looks okay or you'd like any changes made.

File: posts/inside-rust/2023-09-22-project-director-nominees.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+# Announcing the Project Director Nominees
+
+Over the past couple weeks, the Rust Leadership Council has been soliciting nominees from the whole Rust project for the role of Project Director.
+Now that the nomination period is closed, we are happy to announce the list of nominees.
+As described in our [last post], we will elect from these people three new Project Directors.
+
+The nominees are:
+
+- [Jakob Degen](https://github.com/JakobDegen)
+- [Amanieu](https://github.com/Amanieu)
```
Comment (Amanieu) [inline posts/inside-rust/2023-09-22-project-director-nominees.md:17]:
```suggestion
- [Amanieu d'Antras](https://github.com/Amanieu)

```

Comment (eholk) [reply posts/inside-rust/2023-09-22-project-director-nominees.md]: Thanks!

**Comment by thejpster** [comment]
LGTM

File: posts/inside-rust/2023-09-22-project-director-nominees.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+Over the past couple weeks, the Rust Leadership Council has been soliciting nominees from the whole Rust project for the role of Project Director.
+Now that the nomination period is closed, we are happy to announce the list of nominees.
+As described in our [last post], we will elect from these people three new Project Directors.
+
+The nominees are:
+
+- [Jakob Degen](https://github.com/JakobDegen)
+- [Amanieu d'Antras](https://github.com/Amanieu)
+
```
Comment (Mark-Simulacrum) [inline posts/inside-rust/2023-09-22-project-director-nominees.md:18]:
```suggestion
```

**Review by Mark-Simulacrum — APPROVED** [review]
